circle.yml

```diff
@@ -1,91 +1,161 @@
 # Setup in CircleCI account the following ENV variables:
-# IS_PRODUCTION (default: 0)
-# IS_ENTERPRISE (default: 0)
 # PACKAGECLOUD_ORGANIZATION (default: stackstorm)
 # PACKAGECLOUD_TOKEN
-general:
-  # Don't run CI for PR, only for major branches
-  branches:
-    only:
-      - master
-      - /st2-[0-9]+\.[0-9]+\.[0-9]+/
-  build_dir: st2-packages
-  artifacts:
-    - ~/packages
 
-machine:
-  environment:
-    DISTROS: "trusty xenial el6 el7"
-    NOTESTS: "xenial el7"
-    ST2_PACKAGES_REPO: https://github.com/StackStorm/st2-packages
-    # XXX: Set this to 'vX.Y' for release branches
-    ST2_PACKAGES_BRANCH: master
-    # XXX: Set this to 'st2-X.Y.Z' for the release branches
-    ST2MISTRAL_GITREV: master
-    ST2_PACKAGES: "st2mistral"
-    BUILD_DOCKER: 0
-    DEPLOY_DOCKER: 0
-    DEPLOY_PACKAGES: 1
-  ruby:
-    version: ruby-2.0.0-p645
-  pre:
-    - mkdir -p ~/packages
-    # Need latest Docker version for some features to work (CircleCI by default works with outdated version)
-    - |
-      sudo curl -L -o /usr/bin/docker 'http://s3-external-1.amazonaws.com/circle-downloads/docker-1.9.0-circleci'
-      sudo chmod 0755 /usr/bin/docker
-  services:
-    - docker
-    - postgresql
-    - rabbitmq-server
+version: 2
+jobs:
+  # Build st2mistral packages
+  # NB! We can't run mistral tests without st2 being installed, so we just build mistral package here
+  packages:
+    parallelism: 4
+    # 4CPUs & 8GB RAM CircleCI machine
+    # sadly, it doesn't work with 'setup_remote_docker'
+    resource_class: large
+    docker:
+      # The primary container is an instance of the first list image listed. Your build commands run in this container.
+      - image: circleci/python:2.7
+    working_directory: ~/mistral
+    environment:
+      - DISTROS: "trusty xenial el6 el7"
+      - ST2_PACKAGES_REPO: https://github.com/StackStorm/st2-packages
+      - ST2_PACKAGES: "st2mistral"
+      # XXX: Set this to 'vX.Y' for release branches
+      - ST2_PACKAGES_BRANCH: master
+      # XXX: Set this to 'st2-X.Y.Z' for the release branches
+      - ST2MISTRAL_GITREV: master
```

Comment:
We leave the same placeholders as before to make sure that release automation (replacements) is not broken during migration.